--- php-fpm.conf.orig	2018-08-30 23:13:37.773595803 +0200
+++ php-fpm.conf	2018-10-02 11:13:46.043225095 +0200
@@ -21,7 +21,7 @@
 ; into a local file.
 ; Note: the default prefix is /var
 ; Default Value: log/php-fpm.log
-error_log = /var/log/php7.2-fpm.log
+error_log = /proc/1/fd/1
 
 ; syslog_facility is used to specify what type of program is logging the
 ; message. This lets syslogd specify that messages from different facilities
